example/dummy/dummy.go

```diff
@@ -48,13 +49,13 @@ func (app *DummyApplication) Commit() types.Result {
 
 func (app *DummyApplication) Query(query []byte) types.Result {
 	index, value, exists := app.state.Get(query)
-
-	queryResult := QueryResult{index, string(value), exists}
+	queryResult := QueryResult{index, string(value), hex.EncodeToString(value), exists}
 	return types.NewResultOK(wire.JSONBytes(queryResult), "")
 }
 
 type QueryResult struct {
-	Index  int    `json:"index"`
-	Value  string `json:"value"`
-	Exists bool   `json:"exists"`
+	Index    int    `json:"index"`
+	Value    string `json:"value"`
+	ValueHex string `json:"valueHex"`
```

Comment:
I added an additional field to preserve readability for non-hex transactions as well. I don't think it hurts to have both, and it's just the dummy so it is intended solely for humans.
```json
{"index":1,"value":"abc","valueHex":"616263","exists":true}
```